src/MusicalScore/Graphical/MusicSystemBuilder.ts

```diff
@@ -1024,27 +1024,50 @@ export class MusicSystemBuilder {
-                    currentYPosition +=   this.rules.TitleTopDistance + this.rules.SheetTitleHeight +
-                                            this.rules.TitleBottomDistance;
-                }
-                // now add the border-top: everything that stands out above the staffline:
-                currentYPosition += -currentSystem.PositionAndShape.BorderTop;
-                const relativePosition: PointF2D = new PointF2D(this.rules.PageLeftMargin + this.rules.SystemLeftMargin,
-                                                                currentYPosition);
-                currentSystem.PositionAndShape.RelativePosition = relativePosition;
-                // check if the first system doesn't even fit on the page -> would lead to truncation at bottom end:
-                if (currentYPosition + currentSystem.PositionAndShape.BorderBottom > this.rules.PageHeight - this.rules.PageBottomMargin) {
-                    // can't fit single system on page, maybe PageFormat too small
-                    timesPageCouldntFitSingleSystem++;
-                    if (timesPageCouldntFitSingleSystem <= 4) { // only warn once with detailed info
-                        console.log(`warning: could not fit a single system on page ${currentPage.PageNumber}` +
-                            ` and measure number ${currentSystem.GraphicalMeasures[0][0].MeasureNumber}.
-                            The PageFormat may be too small for this sheet."
-                            Will not give further warnings for all pages, only total.`
-                        );
+                if (this.graphicalMusicSheet.MusicPages.length === 1) {
+                    let maxLineCount: number = this.graphicalMusicSheet.Composer?.TextLines?.length;
+                    let maxFontHeight: number = this.graphicalMusicSheet.Composer?.Label?.fontHeight;
+                    let lyricistLineCount: number = this.graphicalMusicSheet.Lyricist?.TextLines?.length;
+                    let lyricistFontHeight: number = this.graphicalMusicSheet.Lyricist?.Label?.fontHeight;
+
+                    maxLineCount = maxLineCount ? maxLineCount : 0;
+                    maxFontHeight = maxFontHeight ? maxFontHeight : 0;
+                    lyricistLineCount = lyricistLineCount ? lyricistLineCount : 0;
+                    lyricistFontHeight = lyricistFontHeight ? lyricistFontHeight : 0;
+
+                    let maxHeight: number = maxLineCount * maxFontHeight;
+                    const totalLyricist: number = lyricistLineCount * lyricistFontHeight;
+
+                    if (totalLyricist > maxHeight) {
+                        maxLineCount = lyricistLineCount;
+                        maxFontHeight = lyricistFontHeight;
+                        maxHeight = totalLyricist;
+                    }
+
+                    currentYPosition += this.rules.TitleTopDistance + this.rules.SheetTitleHeight +
+                                            this.rules.TitleBottomDistance + maxHeight;
```

Comment:
here maxHeight shall not be added.
This actually increases the distance of the title to the notes which can be seen in the visual diffs in all cases.